### ISSUE TITLE
No playback of recorded "Remote"-Tracks

### ISSUE THREAD

**Issue by PhilipBanse** [Studio-Link/webapp#40]
Hi,

Maybe I am stupid, but I can´t get recorded "Remote"-Tracks to be played back after the recording. (The 2 speaker tracks get played well.)

During playback the "Remote"-channels show no level meter in their respective mixer; the sound of the "Remote"-Tracks is not routed to the master mixer although the "Master" box is checked in the routing for track 2. 

Any idea?

Thanks.
Philip
![bildschirmfoto 2016-01-06 um 17 33 05](https://cloud.githubusercontent.com/assets/3398542/12148006/4250940a-b49c-11e5-9dc6-e26ee0808a6c.png)
![bildschirmfoto 2016-01-06 um 17 35 46](https://cloud.githubusercontent.com/assets/3398542/12148007/427d8e60-b49c-11e5-96e3-2ef2a384dfd0.png)

**Comment by sreimers** [comment]
Yes this is something related how the plugin works. You just need to disable the plugin. But be carefull you must disable the cross routing between the tracks, too. On reaper you can automate this with a routing snapshot. it also saves the state of the plugins. In the next days I will make a video.

**Comment by PhilipBanse** [comment]
Ah, cool. Thanks!

**Comment by PhilipBanse** [comment]
But the "Allow feedback in routing" option is not saved by snapshots, is it?

**Comment by sreimers** [comment]
IMHO: its not saved by snapshot, because its a project setting. 
But I'm not 100% sure.

**Comment by PhilipBanse** [comment]
Here it is not saved with a snapshot.

But you want to find a way, because deactivating the fx with the "allow feedback in routing“ option actived hurts. Really hurts.

**Comment by sreimers** [comment]
Yes sry, thats why you should be carefull and disable the cross routing between the tracks first! Sry if this was not clear enough in my earlier post.

**Comment by PhilipBanse** [comment]
No, you were perfectly clear, I just clicked the wrong button ;)
